src/elements/content-preview/ContentPreview.js

```diff
@@ -62,6 +62,7 @@ type Props = {
     enableThumbnailsSidebar: boolean,
     features?: FeatureConfig,
     fileId?: string,
+    fileOptions?: Object,
```

Comment:
Is there a type we can introduce?

Reply:
I can try, but this object is defined in the Preview SDK and consists of dynamic keys (file ids), each with its own options object. It doesn't seem ideal to declare and maintain types for other libraries.